mobile/src/main/java/ch/epfl/sdp/mobile/ui/play/PlayScreen.kt

```diff
@@ -26,24 +25,61 @@ fun PlayScreen(
     modifier: Modifier = Modifier,
     contentPadding: PaddingValues = PaddingValues(),
 ) {
+  val buttonState = rememberExpandableFloatingActionButtonState()
+
   Box(modifier = modifier.fillMaxSize().padding(contentPadding)) {
-    NewGameButton(onNewGame = state.onNewGameClick, modifier = Modifier.align(Alignment.BottomEnd))
+    NewGameButton(
+        state = buttonState,
+        onLocalGame = { state.onLocalGameClick() },
+        onRemoteGame = { state.onOnlineGameClick() },
+        modifier = Modifier,
+    )
   }
 }
 
 /**
  * Composable that composes a new game button
- * @param onNewGame callable upon button action
+ * @param onLocalGame callable upon button action
  * @param modifier the [Modifier] for this composable.
```

Comment:
Miss some parameters in the documentation

Reply:
Fixed in [299fed9](https://github.com/epfl-SDP/android/pull/226/commits/299fed93af87d46ef1db9bbccd46a231460ea110)